src/test/librbd/test_Groups.cc

```diff
@@ -70,6 +70,41 @@ TEST_F(TestGroup, group_createPP)
   ASSERT_EQ(0U, groups.size());
 }
 
+TEST_F(TestGroup, group_get_id)
+{
+  rados_ioctx_t ioctx;
+  rados_ioctx_create(_cluster, _pool_name.c_str(), &ioctx);
+  BOOST_SCOPE_EXIT(ioctx) {
+    rados_ioctx_destroy(ioctx);
+  } BOOST_SCOPE_EXIT_END;
+
+  librbd::RBD rbd;
+  ASSERT_EQ(0, rbd_group_create(ioctx, "group_get_id"));
+  
+  size_t size = 0;
+  ASSERT_EQ(-ERANGE, rbd_group_get_id(ioctx, "group_get_id", NULL, &size));
+  ASSERT_GT(size, 0);
+
+  char id[32];
+  ASSERT_EQ(0, rbd_group_get_id(ioctx, "group_get_id", id, &size));
```

Comment:
This doesn't check the contents of `id` buffer at all.  I'd add something like `ASSERT_EQ(strlen(id) + 1, size);` to validate it at least partially.